lambda/layer/wrapper

```diff
@@ -12,7 +12,7 @@ if [ -z "${OTEL_RUBY_INSTRUMENTATION_NET_HTTP_ENABLED}" ]; then
     export OTEL_RUBY_INSTRUMENTATION_NET_HTTP_ENABLED=false;
 fi
 
-export LAMBDA_RESOURCE_ATTRIBUTES="cloud.region=$AWS_REGION,cloud.provider=aws,faas.name=$AWS_LAMBDA_FUNCTION_NAME,faas.version=$AWS_LAMBDA_FUNCTION_VERSION";
+export LAMBDA_RESOURCE_ATTRIBUTES="cloud.region=$AWS_REGION,cloud.provider=aws,faas.name=$AWS_LAMBDA_FUNCTION_NAME,faas.version=$AWS_LAMBDA_FUNCTION_VERSION,faas.instance=$AWS_LAMBDA_LOG_STREAM_NAME";
```

Comment:
`AWS_LAMBDA_LOG_STREAM_NAME` may not be available during Lambda SnapStart. 
https://docs.aws.amazon.com/lambda/latest/dg/configuration-envvars.html
Will it cause any trouble to the export?

Reply:
If `AWS_LAMBDA_LOG_STREAM_NAME` is missing, then it won't be part of resource attributes. Otel ruby will remove all key with empty value.